shared/components/FormField/FormField.jsx

```diff
@@ -170,9 +185,10 @@ class FormField extends React.Component {
 FormField.propTypes = {
   label: PropTypes.string.isRequired,
   hint: PropTypes.string,
+  hintPosition: PropTypes.oneOf(['inline', 'below']),
   value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
   feedback: PropTypes.oneOf(['success', 'error']),
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
Use `componentWithName` as mentioned above.

Reply:
Leaving it due to reason mentioned [above](https://github.com/telus/tds-core/pull/920#discussion_r259396513).